pkg/apis/org/v1/che_types.go

```diff
@@ -225,67 +203,53 @@ type CheClusterSpecServer struct {
+	// By default, a dedicated plugin registry server is started. When `externalPluginRegistry` is `true`, no such dedicated server
+	// will be started by the Operator and you will have to manually set the `pluginRegistryUrl` field.
 	// +optional
 	ExternalPluginRegistry bool `json:"externalPluginRegistry"`
-	// Map of additional environment variables that will be applied in the generated `che` config map to be used by the Che server,
+	// Map of additional environment variables that will be applied in the generated `che` configMap to be used by the Che server,
 	// in addition to the values already generated from other fields of the `CheCluster` custom resource (CR).
-	// If `customCheProperties` contains a property that would be normally generated in `che` config map from other
-	// CR fields, then the value defined in the `customCheProperties` will be used instead.
+	// When `customCheProperties` contains a property that would be normally generated in `che` configMap from other CR fields,
+	// the value defined in the `customCheProperties` is used instead.
 	// +optional
 	CustomCheProperties map[string]string `json:"customCheProperties,omitempty"`
-	// URL (protocol+hostname) of the proxy server.
-	// This drives the appropriate changes in the `JAVA_OPTS` and `https(s)_proxy` variables
-	// in the Che server and workspaces containers. Only use when configuring a proxy is required.
-	// Operator respects OpenShift cluster wide proxy configuration and no additional configuration is required,
-	// but defining `proxyUrl` in a custom resource leads to overrides the cluster proxy configuration with
-	// fields `proxyUrl`, `proxyPort`, `proxyUser` and `proxyPassword` from the custom resource.
-	// (see the doc https://docs.openshift.com/container-platform/4.4/networking/enable-cluster-wide-proxy.html)
-	// (see also the `proxyPort` and `nonProxyHosts` fields).
+	// URL (protocol+host name) of the proxy server. This drives the appropriate changes in the `JAVA_OPTS` and `https(s)_proxy` variables
+	// in the Che server and workspaces containers.
+	// Only use when configuring a proxy is required. Operator respects OpenShift cluster wide proxy configuration
+	// and no additional configuration is required, but defining `proxyUrl` in a custom resource leads to overrides the cluster proxy configuration
+	// with fields `proxyUrl`, `proxyPort`, `proxyUser` and `proxyPassword` from the custom resource.
+	// (see the doc https://docs.openshift.com/container-platform/4.4/networking/enable-cluster-wide-proxy.html) (see also the `proxyPort` and `nonProxyHosts` fields).
 	// +optional
 	ProxyURL string `json:"proxyURL,omitempty"`
-	// Port of the proxy server. Only use when configuring a proxy is required.
-	// (see also the `proxyURL` and `nonProxyHosts` fields).
+	// Port of the proxy server. Only use when configuring a proxy is required. (see also the `proxyURL` and `nonProxyHosts` fields).
```

Comment:
> (see also the `proxyURL` and `nonProxyHosts` fields).

Looks strange. Maybe:

`See also the `proxyURL` and `nonProxyHosts` fields.`

Reply:
right